### ISSUE TITLE
[Carousel]: QA: The arrow icon which are in enabled state appears as disabled in high contrast themes

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9388]
### Description

Page Element:
Arrow icons on card

Issue Description:
The arrow icon links that are in enabled state but appear as disabled on applying high contrast black & white.

Expected:
Ensure that components appear in defined colors.

User Impact:
When the high contrast standards are not followed, high contrast users will face difficulty in accessing the application.

### Component(s) impacted

Carousel

### Browser

Firefox

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/iframe.html?id=components-carousel--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
